### PR TITLE
[Hotfix] Fixing a level of zoom-related Blob issue

### PR DESCRIPTION
**Pull request by DeityLink**
So ghosts getting forced into blobs get their zoom level reset.

### CODE REVIEW

**Comment by clusterfack** [comment]
This shouldn't be necessary, it should be done in mob/login()

**Comment by clusterfack** [comment]
Also you fucked up in your blob hud update but @gbasood had merged it before I got to comment on it, thats why you encountered this issue in the first place

**Comment by clusterfack** [comment]
It would be rude not to clarify so I'll comment now on that PR to show you how to fix it

**Comment by clusterfack** [comment]
You also forget to have the blob hud reset on login

**Comment by DeityLink** [comment]
- Added support for custom view sizes.
- Fixed blob hud not appearing on login

viewsizes

File: code/game/gamemodes/blob/blobs/core.dm [hunk 1]
```diff
@@ -141,6 +141,7 @@
 		C = new_overmind
 
 	if(C)
+		C.view = world.view
```
Comment (clusterfack) [inline code/game/gamemodes/blob/blobs/core.dm]: You forgot this line

**Comment by DeityLink** [comment]
duh

**Pull request merged**